pkg/cmd/stack/translate.go

```diff
@@ -213,8 +213,46 @@ func translateDeployment(svcName string, s *model.Stack) *appsv1.Deployment {
 	}
 }
 
+func translatePersistentVolumeClaims(name string, s *model.Stack) []apiv1.PersistentVolumeClaim {
+	svc := s.Services[name]
+	result := make([]apiv1.PersistentVolumeClaim, 0)
+	for idx, volume := range svc.Volumes {
+		if volume.LocalPath == "" {
+			continue
+		}
+		volumeSpec := s.Volumes[volume.LocalPath]
+		labels := translateLabels(name, s)
+		for key, value := range volumeSpec.Labels {
+			labels[key] = value
+		}
+
+		pvcName := getVolumeClaimName(name, &volume, idx)
+		pvc := apiv1.PersistentVolumeClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        pvcName,
+				Namespace:   s.Namespace,
+				Labels:      labels,
```

Comment:
We should have a label for first class volumes, something like `"stack.okteto.com/service"`